src/context/App.tsx

```diff
@@ -62,6 +62,7 @@ export const defaultSettings: SettingsState = {
   showAccountHostname: false,
   delayNotificationState: false,
   showPills: true,
+  showNumber: true,
```

Comment:
```suggestion
  showNumber: false,
```
Considering the discussion had, I'd say false is more sensible.

Reply:
I'd rather people opt out like all the other items we show (pills, details notifications)

Reply:
I don't fully understand the reasoning here 🤔 Isn't detailed notifications false by default? Same as `ShowAccountHostname`. There is no real consistency here I think, it is just what we define as being important to be default vs what we don't imo.

Reply:
we changed that in v5.6.0 - https://github.com/gitify-app/gitify/pull/1128

Reply:
`showAccountHostname` was selfishly to help me when rewriting the account module refactoring. It shows when more than 1 account, but for single account users there was no way to control it.  

In time we may make the account header a bit richer, such as:
- profile avatar
- account-specific quick-links.  the current quick-links in the sidebar are a little limited for multiple-account situations.

Reply:
Hmmmmmmm, alright then 🤪 Sorry for being a blocker 😇